### PR TITLE
ASAP-633 APC Submitter's name and Submission Date

### PR DESCRIPTION
**Pull request by gabiayako**
Jira ticket: https://asaphub.atlassian.net/browse/ASAP-633

### CODE REVIEW

File: .github/actions/setup-environment/action.yml [hunk 1]
```diff
@@ -299,9 +299,9 @@ runs:
             echo "crn-auth0-domain=pr-asap-hub.us.auth0.com" >> $GITHUB_OUTPUT
           fi
 
-          CRN_CREATE_ANALYTICS_ALGOLIA_INDEX=${{ contains(github.event.pull_request.labels.*.name, 'crn-create-analytics-algolia-index') || inputs.crn-create-env }}
-          CRN_CREATE_CONTENTFUL_PR_ENV=${{ contains(github.event.pull_request.labels.*.name, 'crn-create-environment') || inputs.crn-create-env }}
-          GP2_CREATE_CONTENTFUL_PR_ENV=${{ contains(github.event.pull_request.labels.*.name, 'gp2-create-environment') || inputs.gp2-create-env }}
+          CRN_CREATE_ANALYTICS_ALGOLIA_INDEX=${{ contains(github.event.pull_request.labels.*.name, 'crn-create-analytics-algolia-index') || inputs.crn-create-env || false }}
```
Comment (gabiayako) [inline .github/actions/setup-environment/action.yml:302]: same thing as https://github.com/yldio/asap-hub/pull/4404, just adding it here as well so the PR is not blocked

File: packages/react-components/src/templates/ManuscriptForm.tsx [hunk 2]
```diff
@@ -703,6 +720,54 @@ const ManuscriptForm: React.FC<ManuscriptFormProps> = ({
...
+            'Already submitted' ? (
+              <>
+                <Controller
+                  name="versions.0.submitterName"
+                  control={control}
+                  rules={{
+                    required: "Please enter the submitter's name.",
+                    maxLength: {
+                      value: 256,
+                      message: 'This title cannot exceed 256 characters.',
```
Comment (AkosuaA) [inline packages/react-components/src/templates/ManuscriptForm.tsx:733]: Maybe "Submitter's name cannot exceed 256 characters."?

File: packages/react-components/src/organisms/ManuscriptVersionCard.tsx [hunk 3]
```diff
@@ -269,11 +270,27 @@ const ManuscriptVersionCard: React.FC<ManuscriptVersionCardProps> = (
                         <Divider />
                       )}
                       <li css={additionalInformationEntryStyles}>
-                        <strong>Requesting APC Coverage?</strong>
+                        <strong>Requested APC Coverage?</strong>
                         <span css={additionalInformationValueStyles}>
                           {version.requestingApcCoverage}
                         </span>
                       </li>
+                      <li css={additionalInformationEntryStyles}>
```
Comment (AkosuaA) [inline packages/react-components/src/organisms/ManuscriptVersionCard.tsx:278]: Should be conditional like the submissionDate
Screenshot 2024-10-07 at 13 46 35

**Review by AkosuaA — APPROVED** [review]
Minor suggestions

File: packages/react-components/src/organisms/ManuscriptVersionCard.tsx (hunk 3 above)
Comment (gabiayako) [reply packages/react-components/src/organisms/ManuscriptVersionCard.tsx:278]: nice catch, thanks!

File: packages/react-components/src/templates/ManuscriptForm.tsx (hunk 2 above)
Comment (gabiayako) [reply packages/react-components/src/templates/ManuscriptForm.tsx:733]: Thanks! I'll change that!

**Pull request merged**